modules/web-mercator/src/web-mercator-viewport.d.ts

```diff
@@ -156,5 +158,8 @@ export default class WebMercatorViewport {
    *    [x, y] measured in pixels.
    * @returns {WebMercatorViewport}
    */
-  fitBounds(bounds, options?: {padding?: number; offset?: [number, number]}): WebMercatorViewport;
+  fitBounds(
+    bounds: [[number, number], [number, number]],
+    options?: Omit<FitBoundsOptions, 'width' | 'height' | 'bounds'>,
```

Comment:
Interesting. 

Not quite sure about defining a type in a "subtractive" way like this, seems (logically) backwards. Would prefer to be adding props to a "smaller" type.

But good enough for now.